src/ngraph_sgcompiler_utils.h

```diff
@@ -48,6 +48,28 @@ inline const ngraph::element::Type& getType(int type) {
   return *ngraphType->second;
 }
 
+inline int getType(const ngraph::element::Type& et) {
+  if (et == ngraph::element::f32) {
+    return mshadow::kFloat32;
+  } else if (et == ngraph::element::f64) {
+    return mshadow::kFloat64;
+  } else if (et == ngraph::element::u8) {
+    return mshadow::kUint8;
+  } else if (et == ngraph::element::i8) {
+    return mshadow::kInt8;
+  } else if (et == ngraph::element::i32) {
+    return mshadow::kInt32;
+  } else if (et == ngraph::element::i64) {
+    return mshadow::kInt64;
+  } else if (et == ngraph::element::boolean) {
+    return mshadow::kUint8;
+  } else {
+    std::cout << et << std::endl;
+    throw std::runtime_error(
+        "NGRAPH_BRIDGE: ngraph element type not supported by NDAarray");
```

Comment:
It seems strange that we're sending part of this error information to `cout`, and the rest as the text of a `runtime_error`.  Should these be unified?  E.g., 
``` c++
std::ostringstrem os;
os << "NGRAPH_BRIDGE: ngraph element type " << et << " not supported by NDArray");
```